scripts/igor-compute_pgen

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+args=("$@")
+species=${args[0]} #human
+chain=${args[1]}  #beta
+seque=${args[2]}
+export tmp_dir=$(mktemp -d)
+export tmp_seqfile=$(mktemp ${tmp_dir}/igorSeq.XXXXXXXXXXXXXXXXXXXXXX.txt)
+echo ${seque} > ${tmp_seqfile}
+export randomBatch=`head /dev/urandom | tr -dc A-Za-z0-9 | head -c10`
+echo ${species}" "${chain}" "${tmp_dir}" "${tmp_seqfile}
+igor -set_wd ${tmp_dir} -batch ${randomBatch} -read_seqs ${tmp_seqfile}
+igor -set_wd ${tmp_dir} -batch ${randomBatch} -species ${species} -chain ${chain} -align --all
+igor -set_wd ${tmp_dir} -batch ${randomBatch} -species ${species} -chain ${chain} -evaluate -output --Pgen
+echo ""
+echo "#################    PGEN   #######################"
+#cat ${tmp_dir}/aligns/${randomBatch}_indexed_sequences.csv
+#cat ${tmp_dir}/aligns/${randomBatch}_indexed_CDR3.csv
+cat ${tmp_dir}/${randomBatch}_output/Pgen_counts.csv
```

Comment:
Since you only have one sequence wouldn't you want to give only the Pgen estimate?
Give the specific line
```
head -2 ${tmp_dir}/${randomBatch}_output/Pgen_counts.csv
```
should only spit the second line. You could then pipe it to a `sed` command to return only the value and not the seq_index (which will be 0 anyway)